clients/fides-js/src/lib/consent-types.ts

```diff
@@ -130,21 +130,21 @@ export interface FidesInitOptions {
  * need to change.
  */
 export interface FidesGlobal extends Fides {
+  config?: FidesConfig;
   consent: NoticeConsent;
   experience?: PrivacyExperience | EmptyExperience;
-  geolocation?: UserGeolocation;
+  fides_meta: FidesJSMeta;
   fides_string?: string | undefined;
+  geolocation?: UserGeolocation;
+  identity: FidesJSIdentity;
+  initialized: boolean;
```

Comment:
Not sure what the intended order here is, it's alphabetish right now.

Reply:
Yeah, I know. I'm doing small tidy ups here and there - personally I think the right order should be:
* properties (alphabetized)
* functions (alphabetized)